### ISSUE TITLE
Откат на оригинальную прошивку SonOff

### ISSUE THREAD

**Issue by i81** [tretyakovsa/Sonoff_WiFi_switch#28]
Друзья, а кто может подсказать как откатиться на стоковую прошивку SonOff? Очень не хватает таймера обратного отсчёта, а он в оригинале есть...

**Comment by tretyakovsa** [comment]
Взять стоковую прошивку и залить :) или дождаться таймера обратного отчета у нас. Если есть идеи как его удобнее оформить в интерфейсе пишите.

**Comment by i81** [comment]
Так,  а чего фантазировать? - начните с http запроса, а интерфейс уже и позднее можно придумать...

**Comment by i81** [comment]
А где "взять" стоковую прошивку?

**Comment by renat2985** [comment]
@i81 насколько я знаю ее не существует в свободном доступе. :)

**Comment by i81** [comment]
От это сюрприз:( игла вся надежда на Вас, что Вы сделаете таймер обратного отсчета...

**Comment by alexandersciv** [comment]
**@renat2985**  Получилось реализовать таймер обратного отчёта?

**Comment by renat2985** [comment]
@alexandersciv, нет. От таймеров в ближайшем будущем вообще избавимся. Время включения\выключения можно будет настроить в сценариях. О обратном отсчете пока следует забыть.

**Comment by tretyakovsa** [comment]
Почему забыт?. Как раз перенос таймеров, через алгоритм сценариев создаст возможность сделать обратный отчет для любых устройств.

**Comment by renat2985** [comment]
@tretyakovsa А как ты это предлагаешь сделать? Выставляем время включения, а каким образом и где указываем продолжительность работы? 
![2017 10 16-22 48 57](https://user-images.githubusercontent.com/1230815/31632040-3c834ca2-b2c4-11e7-808c-5ff1c279555e.png)

**Comment by tretyakovsa** [comment]
Таймеры будут на отдельной страницы как и сейчас. Конечно будут отличия в интерфейсе от текущего варианта. Но вот использовать мы будем один алгоритм разбора сценариев и таймеров и там же будет функционал взаимодействия модулей. Он будет скрыт от пользователя, но все эти функции будут в  одном блоке.

**Comment by renat2985** [comment]
@tretyakovsa может все таки все стоит разместить на 1 странице? Мне сложно с списком сценариев работать, т.к. он не в json формате, а тут еще и разить его нужно убит на то что показывать а что не показывать. Это проблема.

**Comment by i81** [comment]
Так вроде же обсуждали ранее... Ренат, зачем Вы хотите делать время включения и таймер обратного отчёта в одном смысловом поле? "Выставляем время включения, а каким образом и где указываем продолжительность работы?" - это не верный вопрос!
Таймер обратного отсчёта - это не продолжительность работы!!! Таймер обратного отсчёта - это время ЧЕРЕЗ КОТОРОЕ заданное релле перейдет в заданное состояние (переключится, включится или включится)

**Comment by Paly4** [comment]
а на какой Sonoff нужна оригинальная прошивка ?

**Comment by alexandersciv** [comment]
Есть решение вопроса как слить стоковую прошивку. Есть база стоковых прошивок. Есть возможность залить стоковую прошивку.
